projectatomic/ai-studio
# feat: Upload model on podman machine - Qemu

### What does this PR do?

This is built over #204 and add the part for qemu. 
This also needs desktop with https://github.com/containers/podman-desktop/pull/5848
When starting a recipe, the model is uploaded to the podman machine for a fast model service loading

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

it resolves #421 

### How to test this PR?

1. start a recipe, when you see the app is started, open the sample app in the browser, you should be able to play with it without waiting 2-3mins that the model gets loaded.
